### ISSUE TITLE
tests of KeyError after positions calibration_elements are added to DB

### ISSUE THREAD

**Issue by tobiaskleiner** [gammasim/simtools#1051]
In `export_array_elements_as_table`  in `array_model`.

**Comment by GernotMaier** [comment]
@tobiaskleiner - I guess you mean this function here? https://github.com/gammasim/simtools/blob/f92147265e5da546370b0a46c86360047a8e3b57/simtools/model/array_model.py#L368
(Always better to add a permanent link to issues.

Where exactly do you want to add a test for KeyError? Or is it already implemented and this issue is obsolete?

**Comment by tobiaskleiner** [comment]
@GernotMaier the current implementation doesn't export the coordinates of other array elements, only telescope model positions. Maybe we can generalize this.?
Then the KeyError test is not really needed anymore as there is the try ... except now.

**Comment by GernotMaier** [comment]
Yes, this should be generalised (possibly with the option of given a list of array element types in the call). 

But do I see correctly the array_model only knows about telescopes as array elements, but about the calibration devices?

**Comment by tobiaskleiner** [comment]
Yes this needs to be added if we want to output all elements.
